File: src/main/java/me/yoursole/ctf/Commands/Scores.java
package me.yoursole.ctf.Commands;

import me.yoursole.ctf.DataFiles.GameData;
import org.bukkit.Bukkit;
import org.bukkit.ChatColor;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

import java.util.Map;
import java.util.UUID;

public class Scores implements CommandExecutor {
    @Override
    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {

        for(Map.Entry<UUID, Integer> e : GameData.scores.entrySet()){
            sender.sendMessage(ChatColor.GREEN+ Bukkit.getPlayer(e.getKey()).getDisplayName()+": "+e.getValue()+" points");
        }

        return true;
    }
}

File: src/main/kotlin/me/yoursole/ctf/commands/Scores.kt
package me.yoursole.ctf.commands

import com.mojang.brigadier.CommandDispatcher
import me.yoursole.ctf.datafiles.GameData
import net.minecraft.commands.CommandSourceStack
import net.minecraft.commands.Commands.*
import org.bukkit.Bukkit

object Scores : BrigadierCommand {
    override fun register(dispatcher: CommandDispatcher<CommandSourceStack>) {
        dispatcher.register(literal("scores").executes { ctx ->
            ctx.source.bukkitSender.sendMessage(GameData.scores.entries.joinToString("\n") { (uuid, points) ->
                "§a${Bukkit.getPlayer(uuid)?.displayName}: $points points"
            })
            1
        })
    }
}